docs/.vuepress/theme/components/NavLinks.vue

```diff
@@ -45,70 +45,76 @@ export default {
     NavVersionedLink
   },
   computed: {
-    guideLink(){
+    guideLink() {
       return {
         link: '/',
         text: 'Guides'
-      }
+      };
     },
-    examplesLink(){
+    examplesLink() {
       return {
         link: '/examples/',
         text: 'Examples'
-      }
+      };
     },
-    apiLink(){
+    apiLink() {
       return {
         link: '/api/',
         text: 'API Reference'
-      }
```

Comment:
could we turn off the necessity of adding semicolons for Vue components? Vuepress by default does not use semicolons.

Reply:
The current syntax is compatible with Vue wrapper e.g. https://github.com/handsontable/handsontable/blob/master/wrappers/vue/src/BaseEditorComponent.vue. Although I am a fan of semicolons, I have the feeling that common syntax is better for cases where you have to jump between projects/submodules.

Reply:
Ok, let's keep semicolons. (I also prefer using semicolons )

I kept Vue components without them because of vuepress.